lib/ecto/changeset.ex

```diff
@@ -669,6 +669,7 @@ defmodule Ecto.Changeset do
     * `:required` - if the association is a required field
     * `:required_message` - the message on failure, defaults to "can't be blank"
     * `:invalid_message` - the message on failure, defaults to "is invalid"
+    * `:force_update_on_change` - if the update should mark the parent struct as updated
```

Comment:
We should change the description to something like: "force the "parent" schema to be updated in the repository if there is a change in the association, defaults to true".